dev-db/influxdb/influxdb-1.8.10.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module systemd
+GIT_COMMIT=688e697c51fd5353725da078555adbeff0363d01
+GIT_BRANCH=1.8
+
+DESCRIPTION="Scalable datastore for metrics, events, and real-time analytics"
+HOMEPAGE="https://www.influxdata.com"
+
+SRC_URI="https://github.com/influxdata/influxdb/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/Dwosky/packages/raw/main/${P}-deps.tar.xz"
+
+LICENSE="Apache-2.0 BSD BSD-2 EPL-2.0 ISC MIT MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="man"
+
+BDEPEND="man? (
+	app-text/asciidoc
+	app-text/xmlto
+)"
+COMMON_DEPEND="
+	acct-group/influxdb
+	acct-user/influxdb"
+DEPEND="${COMMON_DEPEND}"
+RDEPEND="${COMMON_DEPEND}"
+
+src_compile() {
+	GOBIN="${S}/bin" \
```

Comment:
Uhh, what's up with this?

Reply:
Sorry, mind explaining the issue? Is the dependency (which doesn´t look great right now) or its the way `src_compile` its configured?

I've reused the old ebuild, so there might be items to correct/enhance that I wasn't aware of.

Reply:
It's the `GOBIN="${S}/bin"` part. I'm not super familiar with how go works in Gentoo, but it **looks** like this uses some bundled binary to compile the end-product. Which looks suspicious to be honest. Although I'm not familiar whether `GOBIN` does anything at all with *go-module.eclass*, or whether it's some go-internal thing.

Reply:
According to the documentation, `GOBIN` variable defines the directory where `go install` and `go get` will place binaries after building `main`. I suppose its defined so line 41 gets the binary to install from there.

William Hubs is both maintainer of this package as well as author/maintainer of the _go-module.eclass_, that's why I didn't messed much with the ebuild in the first place.

Reply:
Okay, thanks for explaining. Makes more sense, even though it seems a bit redundant imho (This should be handled by a function outside ebuild).